src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -809,6 +809,20 @@ def _flexible_server_params(command_group):
             c.argument('principal_type', options_list=['--type', '-t'], default='User', arg_type=get_enum_type(['User', 'Group', 'ServicePrincipal', 'Unknown']), help='Type of the Azure AD administrator.')
             c.argument('identity', help='Name or ID of identity used for AAD Authentication.', validator=validate_identity)
 
+        # server advanced threat protection settings
+        for scope in ['update', 'show']:
+            argument_context_string = '{} flexible-server advanced-threat-protection-setting {}'.format(command_group, scope)
+            with self.argument_context(argument_context_string) as c:
+                c.argument('resource_group_name', arg_type=resource_group_name_type)
```

Comment:
Is this only applicable in the context of the current subscription? (no subscription param)
Also, on az sql there's an option to specify '--ids' which enables some sort of bulk operations by calling show/ update on multiple resource ids: [az-sql-server-advanced-threat-protection-setting](https://learn.microsoft.com/en-us/cli/azure/sql/server/advanced-threat-protection-setting?view=azure-cli-latest#az-sql-server-advanced-threat-protection-setting-update:~:text=default%20value%3A%20False-,%2D%2Dids,-One%20or%20more)

Reply:
Updated help